### DISCUSSION TITLE
How to use the Cloze deletions on flashcards

### DISCUSSION THREAD

**Discussion by terrorkota** [st3v3nmw/obsidian-spaced-repetition#205] [Q&A]
I seem to be having an issue with creation of cloze deletion flashcards. I may have confused myself but can’t seem to be able to create them through the method described.

say below was my flashcards:

‘#flashcards

Quote from Eric Hobsbawm

“ended in a =global disorder= whose nature was unclear, and without an =obvious mechanism= for either =ending it or keeping it= under control.”’

Should this make a Cloze Deletion flashcards, if not can you help point me in the right direction how to?

Other than that the plug-in is incredible and have been using it every single day
Cheers and thanks for the help

**Comment by st3v3nmw** [comment]
Hey, so there's a bug with cloze cards. They require a newline after them so that they can be recognized. That might be the issue here. If it's not, please let me know.

**Reply by terrorkota** [reply]
So what would a successfully formatted Cloze card look like? Do I hit enter after the highlighted word? 

Could you send an example?

**Reply by st3v3nmw** [reply]
Like so (there's a blank line after the card):
```
this is a test, ==cloze== goes ==here== & ==here==

```
